R/helpers.R

```diff
@@ -402,7 +405,8 @@ get_hover <- function(hn, data) {
   for(ii in seq_len(ncol(data)))
     data[[ii]] <- format(data[[ii]])
 
-  hdict <- lapply(seq_along(hn), function(ii) list(hn[ii], paste("@", hn2[ii], sep = "")))
+  hdict <- lapply(seq_along(hn), function(ii) list(gsub("hover_", "", hn[ii]), 
```

Comment:
The `hover_` prefix text should not show up in the actual hover tooltip.  It is simply the name of the variable in the spec that holds the hover information.  If it is showing up for you, it's a bug that should be fixed in which case please share an example.

The purpose of the `hover_` prefix is to create a new variable in the data set that contains the hover data text formatted as a string and separate from potentially other versions of the same variable that might be used for any of the actual plotted data.  Thus we want to keep it as a separate column with a different name (the prefix).

For example, the following plot:

```
figure() %>%
  ly_points(Sepal.Length, Sepal.Width, data = iris,
    color = Species, glyph = Species,
    hover = list(Sepal.Length, Sepal.Width))
```

has in its json spec the following:

```
      "tooltips": [
        [
          "Sepal.Length",
          "@hover_Sepal_Length"
        ],
        [
          "Sepal.Width",
          "@hover_Sepal_Width"
        ]
      ]
```

This is telling bokeh to use "Sepal.Length" (no "hover_") as the key and to find the text stored in `@hover_Sepal_length` to use as the value.